### ISSUE TITLE
Using requestContext in dispatch script

### ISSUE THREAD

**Issue by sfraigneau** [microcks/microcks#681]
Hi,

Do you plan to provide a requestContext to the soapui groovy script when evaluated ?

I wonder if we could use the soapui properties substitution in the response to create dynamic results.
I read that it could be possible to create dynamics responses with XPATH but that does not fit our need.

In a script we can do :

```
requestContext.myParam= 1
```

And in the response definition, we refer to the parameter as follow 

```
<something>${myParam}</something>
```

The response received from the server would be

```
<something>1</something>
```

Thank you

**Comment by lbroudoux** [comment]
This may be related to the discussion we had in #621 : usage of a request context that is either fed via `SCRIPT` dispatching, notation in templates or others.

**Comment by lbroudoux** [comment]
Now available on branch `1.7.x` and within the `quay.io/microcks/microcks:next` container image 😉 

You can use some dispatcher script like:
```groovy
import com.eviware.soapui.support.XmlHolder
def holder = new XmlHolder( mockRequest.requestContent )
def name = holder["//name"]

if (name == "Andrew"){
    requestContext.name = "Andrew"
    return "Andrew Response"
} else if (name == "Karla"){
    requestContext.name = "Karla"
    return "Karla Response"
} else {
    return "World Response"
}
```

and use template like this for response:

```xml
<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:hel="http://www.example.com/hello">
   <soapenv:Header/>
   <soapenv:Body>
      <hel:sayHelloResponse>
         <sayHello>Hello ${name} !</sayHello>
      </hel:sayHelloResponse>
   </soapenv:Body>
</soapenv:Envelope>
```
